api/r/CellCensus/vignettes/census_datasets.Rmd

```diff
@@ -76,8 +76,8 @@ expt_query <- tiledbsoma::SOMAExperimentAxisQuery$new(
   census_data$get("mus_musculus"), "RNA",
   obs_query = obs_query
 )
-dataset_seurat <- expt_query$to_seurat(c(counts = "raw"))
-print(dataset_seurat)
+# dataset_seurat <- expt_query$to_seurat(c(counts = "raw"))
```

Comment:
@mlin these two lines prevent pkgdown to building the vignettes. Can you verify if they run for you? My assumption is that those reference a version of tiledbsoma that doesn't have `to_seurat`.

Reply:
@ebezzi please try the following (in an R session) to install the bleeding-edge tiledb & tiledbsoma packages from r-universe:

```
install.packages(c("tiledb","tiledbsoma"), repos = "https://tiledb-inc.r-universe.dev")
```

Reply:
That worked, thanks!